app/Resources/assets/scss/app.scss

```diff
@@ -39,288 +45,264 @@ i {
 /* Page elements
    ------------------------------------------------------------------------- */
 body {
-    display: -webkit-flex;
-    display: flex;
-    -webkit-flex-direction: column;
-    flex-direction: column;
-    min-height: 100vh
+  display: -webkit-flex;
```

Comment:
yeah, we could consider that we target modern browsers only as we target developers. But I would still consider it useful to demo how autoprefixer can be used (many sites will actually need it)